src/instructlab/configuration.py

```diff
@@ -416,30 +416,65 @@ class _mmlu(BaseModel):
     )
 
 
+class _dkbench(BaseModel):
+    """Class describing configuration of DK-Bench evaluation benchmark."""
+
+    judge_model: str = Field(
+        default_factory=lambda: DEFAULTS.JUDGE_MODEL_DK,
+        description="Judge model for DK-Bench.",
+    )
+    input_questions: Optional[str] = Field(
+        default=None,
+        description="File with questions and reference answers used for evaluation during DK-Bench. The file must be valid a '.jsonl' file with the fields 'user_input' and 'reference' in each entry",
+    )
+    temperature: float = Field(
+        default=0.0,
+        description="Temperature for model getting responses during DK-Bench. Temperature controls the randomness of the model's responses. Lower values make the output more deterministic, while higher values produce more random results.",
+    )
+    output_file_formats: str = Field(
+        default_factory=lambda: DEFAULTS.DK_BENCH_OUTPUT_FILE_FORMAT,
+        description="Comma-separated list of file formats for results of the DK-Bench evaluation. Ex: 'csv,jsonl'. Valid options in the list are csv, jsonl, and xlsx. If this option is not provided the results are written as a .jsonl file",
+    )
+    output_dir: str = Field(
+        default_factory=lambda: DEFAULTS.DK_BENCH_DATA_DIR,
+        description="Directory where DK-Bench evaluation results are stored.",
+    )
+
+
 class _mtbench(BaseModel):
     """Class describing configuration of MTBench evaluation benchmark."""
 
     judge_model: str = Field(
         default_factory=lambda: DEFAULTS.JUDGE_MODEL_MT,
-        description="Judge model for mt_bench and mt_bench_branch.",
-    )
-    output_dir: str = Field(
-        default_factory=lambda: DEFAULTS.EVAL_DATA_DIR,
-        description="Directory where evaluation results are stored.",
+        description="Judge model for MT-Bench.",
```

Comment:
ok, I am even more convinced of my earlier comments now. If we are adding duplicate config fields that can be read from based on a single cmd line flag, I think we want an option per benchmark. While this means more flags, its cleaner internally